docs/debugging-openshift.md

```diff
@@ -113,6 +113,22 @@ There are a number of suspicious looking messages that appear in the openshift l
 
         W1125 14:53:10.035539 04523 rest.go:231] No network settings: api.ContainerStatus{State:api.ContainerState{Waiting:(*api.ContainerStateWaiting)(0xc208b29b40), Running:(*api.ContainerStateRunning)(nil), Termination:(*api.ContainerStateTerminated)(nil)}, RestartCount:0, PodIP:"", Image:"kubernetes/pause:latest"}
 
+
+Vagrant synced folder
+----------------
+
+When using [vagrant synced folder](http://docs.vagrantup.com/v2/synced-folders/) you may encounter
+following errors in OpenShift log:
+
+        E0706 11:29:43.421460    3664 empty_dir.go:322] Expected directory "/data/src/github.com/openshift/origin/openshift.local.volumes/pods/4c390e43-23d2-11e5-b42d-080027c5bfa9/volumes/kubernetes.io~secret/deployer-token-f9mi7" permissions to be: -rwxrwxrwx; got: -rwxrwxr-x
+        E0706 11:29:43.421741    3664 kubelet.go:1114] Unable to mount volumes for pod "docker-registry-1-deploy_default": operation not supported; skipping pod
+        E0706 11:29:43.438449    3664 pod_workers.go:108] Error syncing pod 4c390e43-23d2-11e5-b42d-080027c5bfa9, skipping: operation not supported
+
+This will happen when using our provided Vagrantfile to develop OpenShift with vagrant. One of the reasons
+is that you can't use ACLs on shared directories. The solution to this problem is using different directory
```

Comment:
"The solution to this problem is to use a different directory for volume storage than the one in the synced folder"